OpenMcdf.Ole/OlePropertiesContainer.cs

```diff
@@ -65,7 +65,8 @@ public OlePropertiesContainer(CfbStream cfStream)
 
         Context = new PropertyContext()
         {
-            CodePage = pStream.PropertySet0.PropertyContext.CodePage
+            CodePage = pStream.PropertySet0.PropertyContext.CodePage,
```

Comment:
Oh wait. Should Context even be a member of OlePropertiesContainer? Can the one from PropertySet be used instead? I'm not sure of the reason for interaction, but its seems just like duplication at a glance.

Reply:
Hmm, don't know why it was cloning the context originally, and I didn't think to just use the existing context instance directly on read :-(

The write side at https://github.com/ironfede/openmcdf/blob/319c003b7b71cc232683b3048c4622ef528fa11e/OpenMcdf.Ole/OlePropertiesContainer.cs#L243 is already just using the same context instead of cloning it, so it'd make sense to make them consistent